src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/abi/CallingSequenceBuilder.java

```diff
@@ -102,7 +107,11 @@ private static void verifyBoxBindings(Class<?> expectedReturnType, List<Binding>
         Deque<Class<?>> stack = new ArrayDeque<>();
```

Comment:
I've seen in the code that the verifyUnbox method speaks about a `returnType` parameter - should it be an `outType` for symmetry (since, I guess, we use this more than just for returns - e.g. we use it for args in upcalls).

Reply:
I'll change the name. FWIW, `returnType` refers to the type 'returned' by the chain of operations.